src/Sarif.Driver/Sdk/AnalyzeOptionsBase.cs

```diff
@@ -107,7 +107,17 @@ public abstract class AnalyzeOptionsBase : CommonOptionsBase
 
         [Option(
             "baseline",
-            HelpText = "A Sarif file to be used as baseline")]
+            HelpText = "A SARIF file to be used as baseline.")]
         public string BaselineSarifFile { get; set; }
+
+        [Option(
+            "automationId",
+            HelpText = "An id that will be used in the output SARIF and show in the 'automationDetails.id' under 'run'.")]
+        public string AutomationId { get; set; }
+
+        [Option(
+            "automationGuid",
+            HelpText = "A guid that will be used in the output SARIF and show in the 'automationDetails.guid' under 'run'.")]
```

Comment:
>A guid that will be used in the output SARIF and show in the 'automationDetails.guid' under 'run'. [](http://example.com/codeflow?start=24&length=98)

"A guid that will be persisted to the 'Run.AutomationDetails.Guid' property. See section '3.17.4' of the SARIF specification for more information.

Reply:
added.
thanks!